test/xref2/dune

```diff
@@ -14,8 +14,16 @@
 (cram
  (deps %{bin:odoc} %{bin:odoc_print} %{bin:compile}))
 
+(subdir
+ v407_and_above
+ (cram
+  (deps %{bin:odoc} %{bin:odoc_print} %{bin:compile})
+  (enabled_if
+   (>= %{ocaml_version} 4.07.0))))
```

Comment:
Before 4.07, labels in top-comments were ignored by the loader. This is probably a bug.